pkg/daemon/daemon.go

```diff
@@ -2284,15 +2257,9 @@ func (dn *Daemon) checkStateOnFirstRun() error {
 		return nil
 	}
 
-	if dn.nodeWriter != nil {
-		dn.nodeWriter.Eventf(corev1.EventTypeNormal, "BootResync", fmt.Sprintf("Booting node %s, currentConfig %s, desiredConfig %s", dn.node.Name, state.currentConfig.GetName(), state.desiredConfig.GetName()))
```

Comment:
Sorry, could you remind me why this event write was removed? I don't see it directly associated with the alert